configure.ac

```diff
@@ -110,14 +110,14 @@ AC_SUBST(LEG)
 
 # we need to make sure that shared-mime-info is installed, because we need it
 # for accurate mime-type guess
-GLIB_DEP="glib-2.0 >= 2.35, gio-2.0 >= 2.35, shared-mime-info"
+GLIB_DEP="glib-2.0 >= 2.35, gio-2.0 >= 2.35, gio-unix-2.0 >= 2.35, shared-mime-info"
```

Comment:
this breaks windows support (maybe it is already broken?)

ideally gio-unix would be checked alone and made optional, and the code using it would only be included if it is available. this can be done with conditional macros